langfuse/client.py

```diff
@@ -36,9 +36,6 @@ class Langfuse(object):
 
     def __init__(
         self,
-        public_key: str,
-        secret_key: str,
-        host: Optional[str] = None,
```

Comment:
For backwards compatibility, I'd suggest to keep the constructor arguments (public, secret, host) and make them optional. This applies to the SDK and the langchain callback handler. 

If set, constructor arguments then take precedence over environment variable. Removing the argument leads to unexpected behavior for existing users

Reply:
Yes, it makes sense I have made the changes. And if access keys are not present, we are already showing a warning.